modules/ipi-install-retrieving-the-openshift-installer.adoc

```diff
@@ -6,11 +6,23 @@
 [id="retrieving-the-openshift-installer_{context}"]
 = Retrieving the {product-title} installer
 
-Use the `stable-4.x` version of the installer to deploy the generally
+Use the `stable-4.x` version of the installer based on your selected architecture to deploy the generally
 available stable version of {product-title}:
 
+.For x86_64
 [source,terminal,subs="attributes"]
 ----
 $ export VERSION=stable-{product-version}
-export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}')
+$ export RELEASE_ARCH=x86_64
+export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/$RELEASE_ARCH/ocp/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}')
```

Comment:
Hi @kelbrown20, this should be
`$RELEASE_ARCH****/clients****/ocp/$VERSION`

Example: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.11.0-rc.0/release.txt

Reply:
Also, I think that reporting the same section for the two architectures could be avoided. 

With regard to OKD, the problem on this page is larger and should be discussed elsewhere IMO. 

Not only the installer URL used in the previous only-x86 version is not the correct one, but also other sections probably.

Reply:
For the installer URL in the previews only-x86 version, I believe that was my mistake when I was rewriting the export command for both architectures. The live 4.10 installer URL in the same section works for me. 

https://docs.openshift.com/container-platform/4.10/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#retrieving-the-openshift-installer_ipi-install-installation-workflow